slashing-protection/src/main/java/tech/pegasys/web3signer/slashingprotection/interchange/InterchangeV5Importer.java

```diff
@@ -140,6 +146,7 @@ private void parseValidator(
 
     final ArrayNode signedAttestationNode = parentNode.withArray("signed_attestations");
     importAttestations(handle, validator, signedAttestationNode);
+    LOG.info("Imported slashing protection data for validator {}", validator.getPublicKey());
```

Comment:
Could be too spammy with 1000s of validators?

Reply:
ah, I suppose we probably want to know which succeeded though

Reply:
On second thought a full record of successful and failed validators is probably really useful

Reply:
Actually less spammy than it was before now I have hidden the other logs at debug level. Think it's probably ok wouldn't want more than one line of logging though.